DQM/TrackingMonitorSource/plugins/StandaloneTrackMonitor.cc

```diff
@@ -127,15 +223,93 @@ class StandaloneTrackMonitor : public DQMEDAnalyzer {
   MonitorElement* trueNIntH_;
 
   // Exclusive Quantities
-  MonitorElement* nHitsVspTH_;
-  MonitorElement* nHitsVsnVtxH_;
-  MonitorElement* nHitsVsEtaH_;
-  MonitorElement* nHitsVsCosThetaH_;
-  MonitorElement* nHitsVsPhiH_;
```

Comment:
why are these removed?

Reply:
We changed them to nValidHitsvsnVtxH_, etc.